### DISCUSSION TITLE
Centralized management of multiple tenants' data

### DISCUSSION THREAD

**Discussion by alex21145** [django-tenants/django-tenants#937] [Q&A]
How to implement a centralized admin panel (CRUD) that allows to view and manage data from multiple tenants at once? For example, each tenant has a `users` table. We want to be able to bulk edit `users` in all tenants.

**Comment by bgunebakan** [comment]
Did you check [Django-Tenant-users](https://github.com/Corvia/django-tenant-users) package? It provides users table in public schema so you can access all users from different tenants.

**Reply by alex21145** [reply]
The `users` table was just an example; we need a solution for all tenant-specific tables, such as `comments`.

**Comment by alex21145** [comment]
Update: Our customers are under data sovereignty regulations that specify where the physical machines storing data must reside. That's why we must use separate databases.
